action.php

```diff
@@ -68,27 +67,36 @@ protected function html()
         global $INPUT;
         global $USERINFO;
         $id = $INPUT->str('id');
+        $ackSubmitted = $INPUT->str('ack') === 'true';
```

Comment:
I usually prefer to transfer booleans as `0` or `1` in the URL and then use `$INPUT->bool('ack')` to have a real boolean in code.

Reply:
ok, done